src/quo/components/profile/collectible_list_item/view.cljs

```diff
@@ -67,108 +60,129 @@
            {:size   :paragraph-1
             :weight :semi-bold
             :style  style/card-detail-text}
-           collectible-name]]
+           collectible-name]])
 
-         :else
-         [:<>
-          [collection-avatar/view
-           {:size  :size-20
-            :image avatar-image-src}]
-          [rn/view {:style {:width 8}}]
-          [text/text
-           {:size            :paragraph-1
-            :weight          :semi-bold
-            :ellipsize-mode  :tail
-            :number-of-lines 1
-            :style           style/card-detail-text}
-           collectible-name]])])
+   [rn/view
+    {:style (style/avatar-container (:avatar-loaded? state))}
+    (when-not (:avatar-error? state)
+      [:<>
+       [collection-avatar/view
+        {:size        :size-20
+         :on-load-end #(set-state (assoc state :avatar-loaded? true))
+         :on-error    #(set-state (assoc state :error? true))
```

Comment:
I wonder is it better here to do?
`(fn [prev-state] set-state (assoc prev-state :error? true))`

because set-state function will provide the previous state as a param to that function.

cc @flexsurfer, @ulisesmac, @cammellos 
might be worth considering as common practice for other similar cases 👍

Reply:
Done